### PR TITLE
[Merged by Bors] - rename tortoise beacon to beacon

### PR DESCRIPTION
**Pull request by countvonzero**
## Motivation

part of documenting beacons, rename it from tortoise beacon to just beacon

## Changes

- rename it from tortoise beacon to just beacon
- enable beacon weak coin tests in system test 

## Test Plan

unit tests + system tests

## DevOps Notes

- [x] This PR does not require configuration changes (e.g., environment variables, GitHub secrets, VM resources)
- [x] This PR does not affect public APIs
- [x] This PR does not rely on a new version of external services (PoET, elasticsearch, etc.)
- [x] This PR does not make changes to log messages (which monitoring infrastructure may rely on)

### CODE REVIEW

**Review by nkryuchkov — COMMENTED** [review]
Did you not rename the remaining flags (here's the list: `grep -ri 'tortoise\(-\|_\)beacon' .`) intentionally?

File: beacon/beacon.go [hunk 1]
```diff
@@ -69,8 +68,8 @@ type layerClock interface {
 	LayerToTime(types.LayerID) time.Time
 }
 
-// New returns a new TortoiseBeacon.
-func New(
+// NewProtocolDriver returns a new ProtocolDriver.
+func NewProtocolDriver(
```
Comment (dshulyak) [inline beacon/beacon.go]: can we leave as New please? what the point of having verbose initializer if whole module is meant to provide a single initializer?

**Comment by countvonzero** [comment]
> Did you not rename the remaining flags (here's the list: `grep -ri 'tortoise\(-\|_\)beacon' .`) intentionally?

yes. i want to leave that as a separate PR. i don't know for sure if spacecraft configs are still used.

**Review by dshulyak — APPROVED** [review]

**Review by nkryuchkov — APPROVED** [review]

File: beacon/beacon.go (hunk 1 above)
Comment (countvonzero) [reply beacon/beacon.go]: done

**Comment by countvonzero** [comment]
bors merge

**Comment by countvonzero** [comment]
bors merge

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**